README.md

```diff
@@ -492,32 +492,184 @@ ctx.run(q)
 ```
 
 ### joins
+Joins are arguably the largest source of complexity in most SQL queries.
+Quill offers a few different syntaxes so you can choose the right one for your use-case!
 
-In addition to applicative joins Quill also supports explicit joins (both inner and left/right/full outer joins).
+````scala
+// Applicative Joins: 
+A.join(B).on(_.id == _.fk)
+ 
+// Implicit Joins:
+for {
+  a <- A
+  b <- B if (a.id == b.fk) 
+} yield (a, b)
+ 
+// Flat Joins:
+for {
+  a <- A
+  b <- B.join(_.fk == a.id)
+} yield (a, b)
+````
 
-```scala
+Let's see them one by one assuming the following schema:
+````scala
+case class Person(id:Int, name:String)
+case class Address(street:String, zip:Int, fk:Int)
+````
+(Note: If your use case involves lots and lots of joins, both inner and outer. Skip right to the flat-joins section!)
+
+#### applicative joins
 
+Applicative joins are useful for joining two tables together,
+they are straightforward to understand, and typically look good on one line.
+Quill supports inner, left-outer, right-outer, and full-outer (i.e. cross) applicative joins.
+
+```scala
+// Inner Join
+val q = quote {
+  query[Person].join(query[Address]).on(_.id == _.fk)
+}
+ 
+ctx.run(q): List[(Person, Address)]
+// SELECT x1.id, x1.name, x2.street, x2.zip, x2.fk 
+// FROM Person x1 INNER JOIN Address x2 ON x1.id = x2.fk
+ 
+// Left (Outer) Join
+val q = quote {
+  query[Person].leftJoin(query[Address]).on((p, a) => p.id == a.fk)
+}
+ 
+ctx.run(q): List[(Person, Option[Address])]
+// Note that when you use named-variables in your comprehension, quill do it's best to honor them in the query.
```

Comment:
s/it's/its

Reply:
Lol, my wife is a copy editor. Maybe I should ask her to look this over.